pytorch_lightning/accelerators/cpu.py

```diff
@@ -31,6 +31,7 @@ def setup_environment(self, root_device: torch.device) -> None:
             MisconfigurationException:
                 If the selected device is not CPU.
         """
+        super().setup_environment(root_device)
```

Comment:
is this required now?

Reply:
i added it since it overrides the super class, just in case the super class changes its current no-op implementation to something more meaningful later